config/schedule.rb

```diff
@@ -1,8 +1,9 @@
 # frozen_string_literal: true
 env :PATH, ENV['PATH']
 
-# Every minute, check for departure inaccuracies
-every 60.seconds do
+# Every minute after 5am, check for departure inaccuracies
+# raw cron asterisk order: minute, hour, day of month, month, day of week
+every '* 0-2,5-23 * * *' do
```

Comment:
That is until 3:00am.  That is, it runs every minute in hours 0-2, so 2:59, but not 3:00 or 3:01